blockchains/fantom/assets/0x00a35FD824c717879BF370E70AC6868b95870Dfb/info.json

```
@@ -0,0 +1,28 @@
+{
+    "name": "Iron Bank",
+    "website": "https://ib.xyz",
+    "description": "Iron Bank is a decentralized protocol to protocol lending platform. It allows trusted protocols to borrow funds without posting collateral via whitelisting. It is helping build a better and safer DeFi lending ecosystem, by driving capital efficiency as the liquidity infrastructure and backbone for DeFi and CeFi.",
+    "explorer": "https://ftmscan.com/token/0x00a35fd824c717879bf370e70ac6868b95870dfb",
+    "type": "FANTOM",
+    "symbol": "IB",
+    "decimals": 18,
+    "status": "active",
+    "id": "0x00a8b738E453fFd858a7edf03bcCfe20412f0Eb0",
```

The id needs to match with the contract address, also there is not enough token holders
```suggestion
    "id": "0x00a35FD824c717879BF370E70AC6868b95870Dfb",
```